tutorials/io/plot_40_reading_boxy_data.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+r"""
```

Comment:
why is this r here? why is everything commented below?

Reply:
i see you just copied the test_30 one,
this seems strange

so, I am confused why there are still multiple blocks read in, but maybe it can do both single files and multiple blocks so you kept both those functionalities, it may be cleaner to just make it load single files and run loops over multiple files outside of io/boxy.py and append the Raw objects,

was there a reason you decided to keep the multiple files

also I don't think we should ask people to rename their files here, or at least we should indicate what the files names are like for the raw files saved by boxy, we want to make this tutorial as agnostic to the gratton fabiani techniques as we can

Reply:
Not really sure what the 'r' is for or why everything is commented, but the other tutorials are like that too (plot_10, 20, 30). Probably something to do with how the html doc is made.

It will read either single .txt files, or multiple files like it did before, based on what the 'multi_file' argument is (True of False). I kept this is because I thought it would be easier to create a single raw data object rather than having to create multiple raw data objects and merge them. I was mostly worried that merging several raw data objects would cause issues, but since they only contain the raw data, maybe that wouldn't be an issue. But yes, just having it always load a single file would be much more straight forward.....I'll change it to keep things simple.